src/test/java/hudson/plugins/git/util/BuildDataTest.java

```diff
@@ -121,6 +122,39 @@ public void testAddRemoteUrl() {
         assertTrue("Second URL not found in remote URLs", data.getRemoteUrls().contains(remoteUrl2));
         assertEquals(2, data.getRemoteUrls().size());
     }
+    
+    @Test
+    public void testRemoteUrlOrder() {
+    	assertEquals(1, data.getRemoteUrls().size());
```

Comment:
I admit that the formatting in this file is a mess, but I would rather not have different indentation for items at the same scope in the same method.  Could you indent this and the following line by 4 spaces instead of 3 spaces?